README.md

```diff
@@ -58,7 +58,13 @@ yarn add @ant-design/react-native
 ### Link icon fonts
 
 ```bash
-react-native link @ant-design/icons-react-native
+$ npm install react-native link @ant-design/icons-react-native --save
+```
+
+or
+
+```bash
+yarn add react-native link @ant-design/icons-react-native 
```

Comment:
Whats the commands for ?

`npm install react-native ` and `yarn add react-native` ?

Reply:
@bang88 not sure, that's what's already in the docs?

Reply:
The original commands was `react-native link @ant-design/icons-react-native` I think its correct and no need to change it. Thanks anyway.